src/components/MainView/MainPage.jsx

```diff
@@ -127,13 +195,33 @@ export default function VerticalTabs() {
             value={value}
             onChange={handleChange}
           >
-            <StyledTab label="Networking" icon={<HubIcon sx={{ fontSize: '2.5rem' }} />} {...a11yProps(0)} />
-            <StyledTab label="Security" icon={<SecurityIcon sx={{ fontSize: '2.5rem' }} />} {...a11yProps(1)} />
-            <StyledTab label="Collaboration" icon={<GroupsIcon sx={{ fontSize: '3rem' }} />} {...a11yProps(2)} />
-            <StyledTab label="Application Performance" icon={<SpeedIcon sx={{ fontSize: '3rem' }} />} {...a11yProps(3)} />
+            <StyledTab onClick={() => handleTabClick("Networking")} label="Networking" icon={<HubIcon sx={{ fontSize: '2.5rem' }} />} {...a11yProps(0)} />
+            <StyledTab onClick={() => handleTabClick("Security")} label="Security" icon={<SecurityIcon sx={{ fontSize: '2.5rem' }} />} {...a11yProps(1)} />
+            <StyledTab onClick={() => handleTabClick("Collaboration")} label="Collaboration" icon={<GroupsIcon sx={{ fontSize: '3rem' }} />} {...a11yProps(2)} />
+            <StyledTab onClick={() => handleTabClick("Application Performance")} label="Application Performance" icon={<SpeedIcon sx={{ fontSize: '3rem' }} />} {...a11yProps(3)} />
           </StyledTabs>
         </Box>
-        <Box sx={{ flexBasis: '80%', marginTop: '4%', width: `${selectedView === 'Show sankey layout' ? '100%' : 'auto'}` }}>
+        <Box className="custom-box" sx={{ flexBasis: '80%', marginTop: '4%', width: `${selectedView === 'Show sankey layout' ? '100%' : 'auto'}` }}>
```

Comment:
* Since we use [`styled-components`](https://mui.com/system/styled/) to help writing better CSS in React, I recommend creating a component (e.g. `ContentBox`) to address CSS styles defined in both `custom-box` and `sx`. Dynamic styling is also possible, as shown [here](https://dev.to/vtrpldn/styled-components-dealing-with-dynamic-styling-1ja3)

```suggestion
        <ContentBox>
```
* Secondly, I would remove the `margin-top` rule, so the search box goes inline with the top (and the other icons)

Reply:
Thanks Erica, I created the styled component, but I followed [this approach](https://stackoverflow.com/a/69730540) so it is a bit different as I was getting the same error as in the post.